glam/api/management/commands/import_glean_aggs.py

```diff
@@ -11,7 +12,11 @@
 
 
 GCS_BUCKET = "glam-dev-bespoke-nonprod-dataops-mozgcp-net"
-PRODUCT_MODEL_MAP = {"org_mozilla_fenix": "api.FenixAggregation"}
+APP_TO_MODEL = {
+    "org_mozilla_fenix": "api.FenixAggregation",  # nightly
+    "org_mozilla_firefox_beta": "api.FenixAggregation",  # beta
+    "org_mozilla_firefox": "api.FenixAggregation",  # release
```

Comment:
The `app_id`s in the probe scraper conventionally use kebab case: https://github.com/mozilla/probe-scraper/blob/master/repositories.yaml

Reply:
Thanks for pointing that out. The GLAM front-end will be passing this to the API, as well. I think the snake case is fine.